packages/api-reference/src/components/Content/Operation/PathResponses/ExampleResponse.vue

```diff
@@ -97,6 +85,31 @@ const mergeAllObjects = (items: Record<any, any>[]): any => {
         )
       "
       lang="json" />
+    <ScalarCodeBlock
+      v-if="response?.schema['items']?.['allOf']"
+      :content="
+        mergeAllObjects(
+          response?.schema['items']['allOf'].map((schema: any) =>
+            getExampleFromSchema(schema, {
+              emptyString: '…',
+              mode: 'read',
+            }),
+          ),
+        )
```

Comment:
does mergeAllObjects with ciruclar references? 🤔

Reply:
@antlio would you be able to test w circular refs? : )

Reply:
I added a test, works fine!